easybuild/easyconfigs/s/setuptools/setuptools-39.1.0-GCCcore-6.4.0-Python-2.7.14-bare.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'setuptools'
+version = '39.1.0'
+versionsuffix = '-Python-%(pyver)s-bare'
+
+homepage = 'http://pypi.python.org/pypi/setuptools/'
```

Comment:
@geimer this should be reverted to `https`, unless there's a good reason for the change.